### DISCUSSION TITLE
Energy data not loaded

### DISCUSSION THREAD

**Discussion by GH-ChristianHauser** [DarwinsBuddy/WienerNetzeSmartmeter#211] [Q&A] [answered]
Hello everyone,

I would like to thank you in advance for the great project and your work.

I tried to install the integration in Home Assistant and it worked smoothly so far. I then tried to integrate the SmartMeter into the standard energy dashboard. Unfortunately, no data arrives here even after more than 24 hours. Instead, I noticed that unlike the installation screenshots, I only had one entity added (see screenshot 1). I also get the message statistics_not_defined and entity not available in the dashboard (see screenshot 2).

Can you help me here?

Best regards

![image](https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/assets/155384209/b069be79-35e2-4072-9b1d-1be27d6ed45c)
![image](https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/assets/155384209/570e68c1-2c82-468e-ba8d-2049e977d615)

**Comment by manfred2764** [comment]
I also have a problem. the feed-in entity has not been available to me since the update

best regards

**Reply by Kuschla** [reply]
Same for me - it‘s actually available (Attributes of the sensor are shown), but no Data is provided.

At the same time, the _statistics sensors are not created anymore.

**Comment by W-M-B** [comment]
same here. I had the data up and running until it broke. 
I installed the new version but this did not solve the problem. I have an error there:
`Logger: homeassistant.components.sensor
Source: helpers/entity_platform.py:573
Integration: Sensor (documentation, issues)
First occurred: 14. Januar 2024 um 14:19:25 (1 occurrences)
Last logged: 14. Januar 2024 um 14:19:25

wnsm: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 573, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1216, in async_device_update
    await self.async_update()
  File "/config/custom_components/wnsm/statistics_sensor.py", line 80, in async_update
    _sum = Decimal(last_inserted_stat[self._id][0]["sum"])
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
KeyError: 'sum'
`

**Reply by W-M-B** [reply]
I will raise an issue.

**Reply by DarwinsBuddy** [reply, marked as answer]
this should be fixed :) Please reopen a ticket if this still persists
